testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -706,6 +707,11 @@ def GRO():
     def PSFDCD():
         return mda.Universe(PSF, DCD)
 
+    @staticmethod
+    @pytest.fixture()
+    def TPR():
+        return mda.Universe(TPR)
+
```

Comment:
Just to include a CHARMM protein as well:         
```suggestion

    @staticmethod
    @pytest.fixture()
    def PSF():
        return mda.Universe(PSF, DCD)   # load DCD to reduce warnings...
```